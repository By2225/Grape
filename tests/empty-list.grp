fun Int main() {
    List<Int> a;
    a = [];
}
