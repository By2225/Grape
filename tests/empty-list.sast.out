
fun Int main() {
List<Int> a;
(List<Int> : a = (List<*> : []));
return (Int : 0);
}
